examples/khr-3hv/khr-3hv_continuous/controllers/robot_supervisor_manager/KHR3HV_env.py

```diff
@@ -0,0 +1,216 @@
+import numpy as np
+from scipy.ndimage.interpolation import shift
```

Comment:
@tsampazk I think we can use `self.obs[:-self.num_obs] = self.obs[self.num_obs:]` to replace it. We don't need to pad it with NaN since we have `self.obs[-self.num_obs:] = motor_pos` after shifting.
https://github.com/KelvinYang0320/deepworlds/blob/5147c3916d30b46f53242385bd6d3ff3b9ed9626/examples/khr-3hv/khr-3hv_continuous/controllers/robot_supervisor_manager/KHR3HV_env.py#L109-L111

Reply:
Ok that looks good Kelvin, thanks!